server/root

```diff
@@ -1 +0,0 @@
-../
```

Comment:
Why do we no longer need this file?

Reply:
It was causing problems with globs.  `**/*.md` from the root would go into an infinite loop.
Tests are passing without the symlink, let me see if anything else is broken